tests/wpt.js

```diff
@@ -0,0 +1,38 @@
+const WebPageTest = require('webpagetest');
+const fs = require('fs');
+
+const wptServer = 'www.webpagetest.org';
+let wptApiKey = process.env.WPT_API_KEY;
+let wpt = new WebPageTest(wptServer, wptApiKey);
+
+function runWPTTest(url, options = {}) {
+  options.f = 'json';
+  let customMetricsFile = './dist/' + options.label + '.js';
+
+  // check that custom metrics file exists
+  if (!fs.existsSync(customMetricsFile)) {
+    console.error(`Custom metrics file ${customMetricsFile} does not exist`);
+    process.exit(2);
+  } else {
+    options.custom = options.custom || '[_' + options.label + ']\n' + fs.readFileSync('./dist/' + options.label + '.js', 'utf-8');
+  }
+
+  return new Promise((resolve, reject) => {
+    wpt.runTestAndWait(url, options, (error, response) => {
+      console.log(`Running ${options.label} tests for ${url}`);
+      if (error || response.statusCode !== 200) {
+        console.error(`Test for ${url} failed`);
+        console.error(error || response);
+        reject(error || response);
+      } else {
+        console.log(`Test for ${url} succeeded`);
+        console.log(response);
```

Comment:
The response could be huge, not sure if it's helpful to log the whole thing. Maybe just the test ID:

```suggestion
        console.log(`Test ID: ${JSON.parse(response.data.id)}`);
```

Reply:
I kept the test results URL.